Kickstarter-iOS/Features/PledgeView/Controllers/NoShippingPledgeViewController.swift

```diff
@@ -500,6 +506,52 @@ final class NoShippingPledgeViewController: UIViewController,
         self?.viewModel.inputs.scaFlowCompleted(with: status, error: error)
       }
   }
+
+  private func configurePledgeOverTimeUI(_ isPledgeOverTimeAllowed: Bool) {
+    guard isPledgeOverTimeAllowed else { return }
+
+    let collectionPlanStackView = UIStackView(frame: .zero)
+    collectionPlanStackView.axis = .vertical
+    collectionPlanStackView.spacing = Styles.grid(2)
```

Comment:
Moving this into a help method, like the ones we have near the bottom of this class could help simplify this method some.